\section{Presentations}

Most recent slides are available via \httplink[speakerdeck]{www.speakerdeck.com/lcolladotor} or \httplink[slideshare]{www.slideshare.com/lcolladotor}.

\subsection{Talks at conferences}
\cvitem{2018}{\emph{Keynote} From learning to using to teaching to developing R, \emph{CDSBMexico}, Cuernavaca -- MX (remote presentation). \httplink[(slides)]{www.speakerdeck.com/lcolladotor/cdsbmexico}}
\cvitem{2018}{Unique Molecular Correlates of Schizophrenia and Its Genetic Risk in the Hippocampus Compared to Frontal Cortex, \emph{SOBP}, New York -- US.}
\cvitem{2018}{BrainSeq Phase II: Schizophrenia-associated expression differences between the hippocampus and the dorsolateral prefrontal cortex, \emph{BOG}, Cold Spring Harbor -- US.}
\cvitem{2018}{recount workflow: Accessing over 70,000 human RNA-seq samples with Bioconductor, \emph{Online Journal Club by Dennis Lal}, Online. \httplink[(slides)]{www.speakerdeck.com/lcolladotor/recount-webinar}}
\cvitem{2017}{Guiding principles for interactive graphics based on LIBD data science projects, \emph{JSM}, Baltimore -- US. \httplink[(slides)]{www.speakerdeck.com/lcolladotor/jsm-2017}}
\cvitem{2017}{Reproducible RNA-seq analysis with recount2 workshop, \emph{BioC}, Boston -- US. \httplink[(slides)]{www.speakerdeck.com/lcolladotor/bioconductor-2017-recount-workshop}}
\cvitem{2017}{Reproducible RNA-seq analysis with recount2, \emph{ICSA}, Chicago -- US. \httplink[(slides)]{www.speakerdeck.com/lcolladotor/icsa-2017}}
\cvitem{2017}{RNA-seq samples beyond the known transcriptome with derfinder and recount, \emph{SOBP}, San Diego -- US. \httplink[(slides)]{www.speakerdeck.com/lcolladotor/sobp-2017}}
\cvitem{2016}{recount: facilitando el análisis de miles de muestras de RNA-seq, \emph{Genomeeting2016}, Mexico City -- MX. \httplink[(slides)]{www.speakerdeck.com/lcolladotor/genomeeting-2016-mexico}}
\cvitem{2016}{Using Data Science to Study Human Brain Genomic Measurements, \emph{SACNAS}, Long Beach -- US. \httplink[(slides)]{www.speakerdeck.com/lcolladotor/sacnas-2016}}
\cvitem{2016}{\textbf{Collado-Torres L}, et al. Annotation-agnostic differential expression analysis, \emph{ENAR}, Austin -- US. \httplink[(slides)]{www.slideshare.net/lcolladotor/annotationagnostic-differential-expression-analysis}}
\cvitem{2015}{\textbf{Collado-Torres L}, Frazee AC, Love MI, Irizarry RA, Jaffe AE, Leek JT. Annotation-agnostic differential expression analysis, \emph{Genomics and Bioinformatics Symposium}, Center for Computational Genomics, Hopkins, Baltimore -- US.}
\cvitem{2015}{Jaffe AE, Shin J, \textbf{Collado-Torres L}, Leek JT, et al. Dissecting human brain development at high resolution using RNA-seq, \emph{ENAR}, Miami -- US. \httplink[(slides)]{www.slideshare.net/lcolladotor/dissecting-human-brain-development-at-high-resolution-using-rnaseq}}
\cvitem{2014}{Jaffe AE, Shin J, \textbf{Collado-Torres L}, Leek JT, et al. Developmental regulation of human cortex transcription at base-pair resolution, \emph{is3b}: 1st International Summer Symposium on Systems Biology, INMEGEN, Mexico City -- MX.}
\cvitem{2014}{\textbf{Collado-Torres L}, Frazee AC, Love MI, Irizarry RA, Jaffe AE, Leek JT. Fast differential expression analysis annotation-agnostic across groups with biological replicates, LCG 10 year anniversary, LCG-UNAM, Cuernavaca -- MX.}
\cvitem{2013}{\textbf{Collado-Torres L}, Frazee AC, Irizarry RA, Jaffe AE, Leek JT. Differential expression analysis of RNA-seq data at base-pair resolution in multiple biological replicates, \emph{useR2013}, Albacete -- Spain.}
\cvitem{2010}{\textbf{Collado-Torres L}, Reyes-Quiroz A, Cu\'ellar-Partida G, Moreno-Mayar V, Vargas-Ch\'avez C, Collado-Vides J. BacterialTranscription: a R package to identify Transcription Start Sites and Transcription Units, \emph{Bioconductor Developer Meeting}, EMBL, Heidelberg -- Germany.}

\subsection{Posters}
\cvitem{2018}{BrainSeq Phase II: schizophrenia-associated expression differences between the hippocampus and the dorsolateral prefrontal cortex, \emph{SAGES2018}, Philadelphia -- US.}
\cvitem{2017}{\textbf{Collado-Torres L}$\dagger$, Nellore A, Jaffe AE. Getting started with recount2 and accessing it via R, \emph{IDIES2017}, Baltimore -- US.}
\cvitem{2015}{\textbf{Collado-Torres L}, Frazee AC, Love MI, Irizarry RA, Jaffe AE, Leek JT. Annotation-agnostic RNA-seq differential expression analysis software, \emph{ASHG2015} and \emph{IDIES2015}, Baltimore -- US.}
\cvitem{2014}{\textbf{Collado-Torres L}, Frazee AC, Love MI, Irizarry RA, Jaffe AE, Leek JT. Fast annotation-agnostic differential expression analysis, \emph{ENAR} and \emph{Delta Omega Poster Competition (JHBSPH)}, Baltimore -- US.}
\cvitem{2013}{\textbf{Collado-Torres L}, Jaffe AE, Leek JT. Fast annotation-agnostic differential expression analysis, \emph{Genomics and Bioinformatics Symposium}, Center for Computational Genomics, Hopkins, Baltimore -- US.}
\cvitem{2010}{\textbf{Collado-Torres L}, Reyes-Quiroz A, Cu\'ellar-Partida A, Moreno-Mayar V, Taboada B, Vega-Alvarado L, Jim\'enez-Jacinto V, Mendoza-Vargas A, Grande R, Olvera L, Olvera M, Vargas-Ch\'avez C, Júarez K, Collado-Vides J, Morett E. Global Analysis of Transcription Start Sites and Transcription Units in Bacterial Genomes, \emph{From Functional Genomics to Systems Biology}, EMBL, Heidelberg -- Germany.}
\cvitem{2010}{\textbf{Collado-Torres L}, Reyes-Quiroz A, Cu\'ellar-Partida A, Moreno-Mayar V, Taboada B, Vega-Alvarado L, Jim\'enez-Jacinto V, Mendoza-Vargas A, Grande R, Olvera L, Olvera M, Vargas-Ch\'avez C, Júarez K, Collado-Vides J, Morett E. Global Analysis of Transcription Start Sites and Transcription Units in Bacterial Genomes, \emph{BioC2010}, FHCRC, Seattle -- US.}

\subsection{Other talks}
\cvitem{2018}{Reproducible RNA-seq analysis with recount2 and recount-brain, LCG-UNAM via Skype.}
\cvitem{2017}{Reproducible Research and Bioinformatics, \emph{Summer Institute}, JHBSPH, Baltimore -- US. \httplink[(slides)]{www.speakerdeck.com/lcolladotor/reproducibility-2017}}
\cvitem{2016}{Introduction at Kandahar University MPH training event. \httplink[(slides)]{www.speakerdeck.com/lcolladotor/kandahar-university-training-introduction}}
\cvitem{2015}{dbFinder, \emph{Joint Genomic Meeting}, JHBSPH, Baltimore -- US.}
\cvitem{2015}{Easy parallel computing with BiocParallel and HTML reports with knitrBootstrap, \emph{Biostatistics Computing Club}, JHBSPH, Baltimore -- US.}
\cvitem{2015}{Does mapping simulated RNA-seq reads provide information?, \emph{Joint Genomic Meeting}, JHBSPH, Baltimore -- US.}
\cvitem{2014}{Git for research, \emph{Biostatistics Computing Club}, JHBSPH, Baltimore -- US.}
\cvitem{2013}{Introduction to ggbio, \emph{Genomics for Students}, JHBSPH, Baltimore -- US.}
\cvitem{2013}{Introduction to knitr, \emph{Biostatistics Computing Club}, JHBSPH, Baltimore -- US.}
\cvitem{2013}{Introduction to High-Throughput Sequencing and RNA-seq, \emph{Genomics for Students}, JHBSPH, Baltimore -- US.}
\cvitem{2012}{DEXSeq paper discussion, \emph{Genomics for Students}, JHBSPH, Baltimore -- US.}
\cvitem{2012}{Introduction to R and Biostatistics, LCG-UNAM via Skype.}
\cvitem{2012}{Introducing Git while making your academic webpage, \emph{Biostatistics Computing Club}, JHBSPH, Baltimore -- US.}
\cvitem{2011}{Introducing Biostatistics to first year LCG students, LCG-UNAM via Skype.}
\cvitem{2010}{Introduction to using Bioconductor for High Throughput Sequencing Analysis, \emph{National Bioinformatics Week}, CCG-UNAM, Cuernavaca -- MX.}
\cvitem{2009}{Bacteriophages: analyzing their diversity, \emph{LCG third generation symposium}, CCG-UNAM, Cuernavaca -- MX.}
